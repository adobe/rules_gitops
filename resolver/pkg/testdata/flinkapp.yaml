apiVersion: flink.k8s.io/v1beta1
kind: FlinkApplication
metadata:
  name: wordcount-operator-example
spec:
  image: flinkapp-image
