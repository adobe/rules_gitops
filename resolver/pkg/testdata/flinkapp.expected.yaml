apiVersion: flink.k8s.io/v1beta1
kind: FlinkApplication
metadata:
  name: wordcount-operator-example
spec:
  image: docker.io/kube/flink/image:tag
